src/usr/local/www/diag_ping.php

```diff
@@ -55,7 +58,11 @@
 	if (($_REQUEST['count'] < 1) || ($_REQUEST['count'] > MAX_COUNT)) {
 		$input_errors[] = sprintf(gettext("Count must be between 1 and %s"), MAX_COUNT);
 	}
-
+	
+	if (($_REQUEST['wait'] < 1) || ($_REQUEST['wait'] > MAX_WAIT)) {
```

Comment:
You should also test to make sure it's an integer, e.g. `is_numericint($_REQUEST['wait'])`